CONTRIBUTING.md

```diff
@@ -1,13 +1,13 @@
-# Contributing to The GRAPHITTI Project
+# Contributing to The Graphitti Project
 
-Thank you for your interest in the Graphitti Project, which includes the Workbench software and data provenance system. This project operates with an [Apache 2.0 license](../../LICENSE) which provides wide reusability and adaptability, under the proviso of citing the originators and maintaining provenance information.
+Thank you for your interest in the Graphitti Project, which includes the Workbench software and data provenance system. This project operates with an [Apache 2.0 license](LICENSE) which provides wide reusability and adaptability, under the proviso of citing the originators and maintaining provenance information.
 
-## External from the BioComputing Lab
-For people outside of the [UW Bothell Biocomputing laboratory](http://depts.washington.edu/biocomp/) (BCL), we use a [fork and pull development model](https://help.github.com/articles/about-collaborative-development-models/). If you're interested in adapting this project for your own use, then please feel free to make your own copy of this repository and adapt it to your work. We would be greatly interested to learn about what you do, potentially incorporating your work back into this main repository. *Please cite us in your work*; the repository [README](../../Desktop/Graphitti/BG-reorg/README.md) has a DOI for that purpose.
+## External from the Intelligent Networks Lab
+For people outside of the [UW Bothell Intelligent Networks laboratory](http://depts.washington.edu/biocomp/) (INL), we use a [fork and pull development model](https://help.github.com/articles/about-collaborative-development-models/). If you're interested in adapting this project for your own use, then please feel free to make your own copy of this repository and adapt it to your work. We would be greatly interested to learn about what you do, potentially incorporating your work back into this main repository. *Please cite us in your work*; the repository [README](README.md) has a DOI for that purpose.
 
 If you're making modifications that you'd like to be merged into our code base, then please see the Workflow section, below. When unsure, contact us ahead of time.
 
-## For the BioComputing Lab
+## For the Intelligent Networks Lab
 For UW Bothell students interested in working in the BCL, we use a [shared repository development model](https://help.github.com/articles/about-collaborative-development-models/). If you're interested in contributing directly to this project, then please contact [Prof. Michael Stiber](mailto:stiber@uw.edu) and read the information below.
 
 ## Workflow
```

Comment:
The link for "Managing your work on GitHub" doesn't seem correct: https://help.github.com/categories/managing-your-work-on-github/
It sends me to the github issues doc page (https://docs.github.com/en/issues). I am not sure what this link is supposed to be.

Reply:
https://web.archive.org/web/20180708165903/https://help.github.com/categories/managing-your-work-on-github/
This is what it used to be.

Reply:
It seems that this is still a very similar page. In fact, I would say that the current title is not so good. But I'll change the link text...

Reply:
This is exactly what I was about to write. It is the exact same content, just a different title.